server/searchPage.js

```diff
@@ -10,8 +10,9 @@ const pool = new Pool({
 
 
 module.exports.applySearch = async event => {
-		const selectAllCleaners = 
-		"SELECT b.profile_id, b.first_name,	b.middle_name, b.last_name,	b.contact_num, b.app_status, b.email, b.address, b.city, b.province, b.postal_code,	b.health_ins, b.dob, b.current_occup, b.gov_id, d.cert_name, f.is_complete, g.relationship FROM cleaner_profile AS b LEFT JOIN enrolled AS f ON f.profile_id = b.profile_id LEFT JOIN certification AS d ON d.cert_id = f.cert_id LEFT JOIN has_ref AS e ON e.profile_id = b.profile_id LEFT JOIN reference AS g ON g.ref_id = e.ref_id"
+		// const selectAllCleaners = "SELECT * FROM cleaner_profile"
+		const selectAllCleaners = "SELECT b.profile_id, b.first_name,	b.middle_name, b.last_name,	b.contact_num, b.app_status, b.email, b.address, b.city, b.province, b.postal_code,	b.hin, b.dob, b.current_occup, b.gov_id, d.cert_name, f.is_complete, g.relationship FROM cleaner_profile AS b LEFT JOIN enrolled AS f ON f.profile_id = b.profile_id LEFT JOIN certification AS d ON d.cert_id = f.cert_id LEFT JOIN has_ref AS e ON e.profile_id = b.profile_id LEFT JOIN reference AS g ON g.ref_id = e.ref_id"
```

Comment:
Looks like there are tabs used as spaces here (for example, after "b.first_name". For consistency and cleanliness, the spacing should be the same.